### PR TITLE
Daytime cloudcover

### PR DESCRIPTION
**Pull request by JrtPec**
Adds a column to the weather data with an average daily cloud cover, but only for the hours between sunrise and sunset.

Update: Code is now generic, meaning daytime average can now be calculated for every value. Implemented are cloud cover and temperature.

### CODE REVIEW

File: library/forecastwrapper.py [hunk 1]
```diff
@@ -198,10 +208,48 @@ def get_weather_ts(self, date):
...
+            day_data.update({'daytimeTemperature':dtt})
+
         #convert to a single row in a dataframe and return
         daylist = [pd.Series(day_data)]
         return pd.concat(daylist,axis=1).T
 
+    def _get_daytime_avg(self, forecast, key):
+        """
+        Calculate the average for a given value during daytime hours (from sunrise to sunset)
+        :param forecast: Forecast object
```
Comment (saroele) [inline library/forecastwrapper.py]: @JrtPec  looks like you change the docstring formatting here.  Why?

Comment (JrtPec) [inline library/forecastwrapper.py]: It's done automatically by Pycharm…

Comment (saroele) [inline library/forecastwrapper.py]: ah, ok.  I prefer consistency in the code over tool-specific randomness.  We have to set a convention and stick to it. 

I don't know in which direction the python community is evolving regarding docstring style and formatting.  Let's investigate.

**Comment by JrtPec** [comment]
@saroele I have found this list of Python Docstring styles: http://stackoverflow.com/a/24385103/5502568

Turns out there is a setting in Pycharm where you can choose from these styles. So since we were using the Numpy style before, I suggest we stick to that. I also think it's one of the more readable ones.

**Comment by saroele** [comment]
Great, so let's stick to the numpy style for our docstrings.  Do not forget the spaces surrounding the ` : ` because without spaces the documentation does not render nicely (at least in Spyder).

**Pull request merged**